compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/backhandler/DesktopBackGestureDispatcher.kt

```diff
@@ -26,18 +26,18 @@ import androidx.compose.ui.input.key.type
 @OptIn(ExperimentalComposeUiApi::class)
 internal class DesktopBackGestureDispatcher: BackGestureDispatcher() {
     fun onKeyEvent(event: KeyEvent): Boolean {
-        if (event.type == KeyEventType.KeyUp && event.key == Key.Escape) {
-            onBack()
-            return true
+        if (event.type == KeyEventType.KeyDown && event.key == Key.Escape) {
```

Comment:
Why not `KeyUp` as before?

Reply:
I asked few colleagues about expectations and default desktop behavior and found that it should be the down event. And @m-sasha told me the same

Reply:
And, by the way, dialogs are being hidden by the down event too